app/components/docs/components_table.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class Docs::ComponentsTable < ApplicationComponent
+class Components::Docs::ComponentsTable < Components::Base
```

Comment:
you already are on Components directory, it means that you can use only Base make it less verbose

Components::Docs::ComponentsTable < Base

or even better:

```ruby
module Components
  module Docs
    ComponentsTable < Base
    end
  end
end
```